src/test/java/com/hocs/test/glue/decs/ValidationStepDefs.java

```diff
@@ -1321,7 +1314,7 @@ public void iTriggerTheErrorMessageAtTheStage(String errorMessage, String stage)
                                 safeClickOn(cch.completeTheCaseClosePermanentlyRadioButton);
                                 safeClickOn(continueButton);
                                 cch.enterReasonForCaseCompletion();
-                                clickTheButton("Complete case");
+                                clickTheButton("Confirm");
```

Comment:
I'm not too sure this would work, there is an extra radio button to select between entering the reason and confirming.

Reply:
I added the 'Complete case' button click in the enterReasonForCaseCompletion method. Double checked and the validation scenario does pass.
I will rename the method to submitReasonForCaseCompletion, as that will be clearer and match other methods for similar pages.

Reply:
I have updated this method to have the name submitReasonForCaseCompletion.